nss-day-cohort-60/rare-client-tenacious-tators
# Subscribe to User

Closes Ticket # 8


# Description of Proposed Changes

Added subscriptions.js to managers
Added subscribe.js to users director
Added props to UserDetails and application views

Given a user is on another user's detail page
When they click on the subscribe button
Then that subscription should be added
And the user should be routed to the homepage /

---


# Steps to Test

1. Fetch the [subscribe-to-user-MP] branch and switch to it
2. Verify your server is open and running debugger. Open sqlite and view subscriptions table.
3. Login to Rare Publishing in browser
4. Navigate to User Management in browser
5. Click on a user to see their details
6. Click "Subscribe"
7. Back in the server-side code, verify the new subscription now appears in the subscriptions table by refreshing the table
